api/src/Modules/volunteers/Controller/Hours/GetEventHours.php

```diff
@@ -68,8 +68,12 @@ public function buildResource(Request $request, Response $response, $params): ar
         }
 
         $sum = 0;
-        foreach ($data as $entry) {
+        foreach ($data as &$entry) {
             $sum += $entry['hours'] * $entry['modifier'];
+
+            /* some data conversion */
+            $datetime = \DateTime::createFromFormat('Y-m-d H:i:s', $entry['end']);
+            $entry['end'] = $datetime->format(\DateTime::RFC3339);
```

Comment:
Can you give an example of the output from this?

And is there a reason to pick RFC3339 vs ISO-8601 which is more typical? How does this show up on the UI when the data is fetched from the API?

Reply:
The OpenAPI 3.0 spec states  the RFC3339 specification

https://swagger.io/docs/specification/data-models/data-types/#string

Reply:
It seems that RFC3339 is what defined the ISO-8601 standard, so it turns out it's the same. :)

Is there a reason we are formatting this in the API output? Does it not come out of the database this way?

Reply:
No, it seems to come out of the database in 'Y-m-d H:i:s' but we need it in something like `2017-07-21T17:32:28Z` Maybe it is just adding the 'T' and 'Z' but it seemed more correct to do the full format.